core/sql/comexe/ComTdbFastTransport.h

```diff
@@ -395,9 +399,10 @@ class ComTdbFastExtract : public ComTdb
   UInt16       filler_;                                      // 130 - 131
   UInt32       childDataRowLen_;                             // 132 - 135
   Int64        modTSforDir_;                                 // 136 - 143
+  UInt16       hdfsIoByteArraySize_;                         // 144 - 147 
```

Comment:
Let me change it to int. But 64 MB should be good enough because default block size is 64 MB in Apache hadoop. In Cloudera hadoop is changed 128MB. But, this allocation needs to be done in Java and hence restricting to less value than 64 MB would put less burden on the java objects space management in Trafodion.